src/config/icons/Icon.tsx

```diff
@@ -77,8 +69,8 @@ export type IconProps = {
 
 export const IconSet = Tabler;
```

Comment:
I see that the widgets are not arranged in an orderly way, I think it's because of the Tabler icon family. We have encountered this problem before but it was not solved. Maybe now is the time to fix it.

<img width="329" alt="image" src="https://github.com/user-attachments/assets/f0d0abae-2f8a-4362-8914-8929d11fb952">

Reply:
good point! i will investigate

Reply:
check

<img width="329" alt="image" src="https://github.com/user-attachments/assets/4cded8ca-a366-4c7b-9311-7d86bca0fae1">

Reply:
Perfekt 🤩